examples/with-styletron/next.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  webpack: function (config) {
```

Comment:
It's setting `styletron-server` as an external dependency, that way webpack will not add it to the bundle and instead it's going to get it from `node_modules`